sys/net/application_layer/cord/Kconfig

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
```

Comment:
What about `CORD_EP` to configure the endpoint name?

Reply:
CORD_EP by itself is not configurable, hence does it qualify as a CT parameter?

Reply:
But it is:
https://github.com/RIOT-OS/RIOT/blob/2b90d9b83378e20de11c0e2a494a63b7555da206/sys/include/net/cord/config.h#L58-L59

Reply:
Oh okay. I have updated the file. Usually, the macros I came across had a definition with/without default. Since it was not defined in the header, I presumed that the macro was just a symbol to model a dependency.  Thanks for pointing that out.